samples/data-grid/cypress/sorting-options/demo.js

```diff
@@ -0,0 +1,62 @@
+const columnSelectEl = document.getElementById('select-column');
```

Comment:
Do we need this cypres demo? It looks identical to the one in `/basic`- if possible we can reuse it.